### ISSUE TITLE
Console filled with flush messages

### ISSUE THREAD

**Issue by tapz** [mixpanel/mixpanel-iphone#615]
**Integration Method:** CocoaPods
**Xcode Version:** 8.1
**Library Version:** 3.0.6
**Platform:** iOS
**Device:** iPhone 7
**Language:** Swift

[ ] A/B Testing Manually Disabled? (DISABLE_MIXPANEL_AB_DESIGNER)

**Description:** 

Console full of this. New one added every second.

```
Nov 16 19:26:46  MyApp[6266] <Info>: <Mixpanel: 0x10273a390 - Token: ********************************> flush starting
Nov 16 19:26:46  MyApp[6266] <Info>: <Mixpanel: 0x10273a390 - Token: ********************************> archiving events data to /var/mobile/Containers/Data/Application/666AFC11-7893-4D1E-8F36-C04B3E0BD444/Library/mixpanel-********************************-events.plist: (
	)
Nov 16 19:26:46  MyApp[6266] <Info>: <Mixpanel: 0x10273a390 - Token: ********************************> archiving people data to /var/mobile/Containers/Data/Application/666AFC11-7893-4D1E-8F36-C04B3E0BD444/Library/mixpanel-********************************-people.plist: (
	)
Nov 16 19:26:46  MyApp[6266] <Info>: <Mixpanel: 0x10273a390 - Token: ********************************> archiving properties data to /var/mobile/Containers/Data/Application/666AFC11-7893-4D1E-8F36-C04B3E0BD444/Library/mixpanel-********************************-properties.plist: {
	    distinctId = "0A27CBCE-CCCA-4A89-8A0C-802AD17573C5";
	    peopleUnidentifiedQueue =     (
	    );
	    shownNotifications = "{(\n)}";
	    shownSurveyCollections = "{(\n)}";
	    superProperties =     {
	    };
	    timedEvents =     {
	    };
	}
Nov 16 19:26:46  MyApp[6266] <Info>: <Mixpanel: 0x10273a390 - Token: ********************************> flush complete
```

**Expected Behavior:** 

No logging.

**Comment by tapz** [comment]
This started to happen when I updated the MixPanel pod.

**Comment by pchien** [comment]
Yeah sorry about that. It will be fixed in the next release but for now `[[Mixpanel sharedInstance] setEnableLogging:NO]` should turn it off. Let me know it that doesn't work.

**Comment by Benjamin-Dobell** [comment]
Hmm, this is annoying enough that it's probably worth making a minor release (assuming the next major release is still a while off).

**Comment by MattNewberry** [comment]
Agree 100% - This slows down our app significantly given a large log size.

**Comment by yarneo** [comment]
We totally agree and will send out an update this week. Sorry for the inconvenience.

**Comment by pchien** [comment]
We just released 3.0.7 with the fix for this. Sorry for the wait.
